worker/volume_client.go

```diff
@@ -68,11 +68,15 @@ type VolumeClient interface {
 var ErrVolumeExpiredImmediately = errors.New("volume expired immediately after saving")
 
 type ErrCreatedVolumeNotFound struct {
-	Handle string
+	Volume db.CreatedVolume
 }
 
 func (e ErrCreatedVolumeNotFound) Error() string {
-	return fmt.Sprintf("failed to find created volume in baggageclaim. Volume handle: %s", e.Handle)
+	p, j, t, err := e.Volume.TaskIdentifier()
```

Comment:
Hmm I don't think the pipeline/task information will always be correct, since this error could happen for other reasons too (like checking resources or running a `get`/`put` step).

The message should already be in the appropriate context, since the UI will show the error in the output of the step that was running, and the system log lines probably have enough context in the message/session/data.

I'd recommend keeping it shorter and only showing the more critical information.